enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/EnterpriseCoreEditionModule.java

```diff
@@ -45,8 +45,9 @@
 import org.neo4j.causalclustering.identity.MemberId;
 import org.neo4j.causalclustering.load_balancing.LoadBalancingPluginLoader;
 import org.neo4j.causalclustering.load_balancing.LoadBalancingProcessor;
-import org.neo4j.causalclustering.load_balancing.procedure.GetServersProcedureV1;
-import org.neo4j.causalclustering.load_balancing.procedure.GetServersProcedureV2;
+import org.neo4j.causalclustering.load_balancing.procedure.GetServersProcedureForSingleDC;
+import org.neo4j.causalclustering.load_balancing.procedure.LegacyGetServersProcedure;
+import org.neo4j.causalclustering.load_balancing.procedure.GetServersProcedureForMultiDC;
```

Comment:
I don't like the renames from GetServersV1/V2 to specialized versions. The V1 and V2 are external interfaces which shall not change and they are exposed under exactly those names. They represent the contract and one could imagine a V3 in the future, for example to include physical rule tables or metrics. I would rather just keep that scheme for clarity.